entity-service/src/test/kotlin/com/egm/stellio/entity/service/ObservationEventListenerTests.kt

```diff
@@ -0,0 +1,72 @@
+package com.egm.stellio.entity.service
+
+import com.egm.stellio.shared.model.*
+import com.egm.stellio.shared.util.loadSampleData
+import com.egm.stellio.shared.util.toUri
+import com.ninjasquad.springmockk.MockkBean
+import io.mockk.confirmVerified
+import io.mockk.every
+import io.mockk.mockkClass
+import io.mockk.verify
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.test.context.ActiveProfiles
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.NONE, classes = [ObservationEventListener::class])
+@ActiveProfiles("test")
+class ObservationEventListenerTests {
+
+    @Autowired
+    private lateinit var observationEventListener: ObservationEventListener
+
+    @MockkBean(relaxed = true)
+    private lateinit var entityService: EntityService
+
+    @MockkBean(relaxed = true)
+    private lateinit var entityAttributeService: EntityAttributeService
+
+    @MockkBean(relaxed = true)
+    private lateinit var entityEventService: EntityEventService
+
+    private val apicContext = listOf(
```

Comment:
add an APIC_COMPOUND_CONTEXT like has been done for aquac in https://github.com/stellio-hub/stellio-context-broker/blob/develop/shared/src/testFixtures/kotlin/com/egm/stellio/shared/util/TestUtils.kt#L18